src/ImageSequencer.js

```diff
@@ -232,9 +235,12 @@ ImageSequencer = function ImageSequencer(options) {
       str.substr(0,str.indexOf('(')),
       str.substr(str.indexOf('(')+1)
     ]
-    
+    str[1] = str[1].split('%20').join(' ');
+    str[1] = str[1].split(', ').join('|');
```

Comment:
Here are you replacing commas with `|` pipes? I think the encode/decode should make that unnecessary, right?